tests/layer_tests/pytorch_tests/test_mul.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+
+from pytorch_layer_test_class import PytorchLayerTest
+
+
+class TestMul(PytorchLayerTest):
+    def _prepare_input(self):
+        return (self.input_array.astype(self.input_type), self.other_array.astype(self.other_type))
+
+    def create_model(self):
+        import torch
+
+        class aten_mul(torch.nn.Module):
+            def __init__(self):
+                super(aten_mul, self).__init__()
+
+            def forward(self, input_tensor, other_tensor):
+                return torch.mul(input_tensor, other_tensor)
+
+        ref_net = None
+
+        return aten_mul(), ref_net, "aten::mul"
+
+    @pytest.mark.parametrize(("input_array", "other_array"), [
+        [np.random.rand(1, 2), np.random.rand(2, 1)],
+        [np.random.rand(3, 1, 2), np.random.rand(3, 1, 2)],
+        [np.random.rand(4, 1, 1), np.random.rand(1, 1, 4)],
+    ])
+    @pytest.mark.parametrize(("types"), [
+        (np.float32, np.float32),
+        # Type promotion
+        pytest.param((np.int32, np.float32), marks=pytest.mark.xfail),
```

Comment:
can you please add ticket number for `xfail` mark?

Reply:
Done